### ISSUE TITLE
`go get github.com/vmware-tanzu/carvel-vendir` fails

### ISSUE THREAD

**Issue by paulczar** [carvel-dev/vendir#35]
Looks like there's a bug getting the checksum of carvel-vendir due to a file name `aux.go`.

```
❯ go version
go version go1.13.15 linux/amd64

❯ go get github.com/vmware-tanzu/carvel-vendir
go: downloading github.com/vmware-tanzu/carvel-vendir v0.14.0
verifying github.com/vmware-tanzu/carvel-vendir@v0.14.0: github.com/vmware-tanzu/carvel-vendir@v0.14.0: reading https://sum.golang.org/lookup/github.com/vmware-tanzu/carvel-vendir@v0.14.0: 410 Gone

❯ curl https://sum.golang.org/lookup/github.com/vmware-tanzu/carvel-vendir@v0.14.0
not found: create zip: malformed file path "pkg/vendir/config/aux.go": "aux" disallowed as path element component on Windows

```

**Comment by paulczar** [comment]
It looks like it was fixed here, just after the 0.14.0 release, cutting a new release should fix it.

https://github.com/vmware-tanzu/carvel-vendir/commit/393bea7fb7386120edb1170c5dce25b067701183

**Comment by cppforlife** [comment]
cut the new release: https://github.com/vmware-tanzu/carvel-vendir/releases/tag/v0.15.0
